integration-tests/modules/__tests__/customer/store/update-customer.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { medusaIntegrationTestRunner } from "@medusajs/test-utils"
```

Comment:
Can you move this test to `integration-tests/http/__tests__/customer/store/customer.spec.ts` under a new describe i.e. what you have from line 15?

Reply:
legend! 💪🏻